internal/pkg/aws/sqs/queue.go

```diff
@@ -0,0 +1,113 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package sqs provides a client to make API requests to Amazon SQS Service.
+package sqs
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+)
+
+var SQSQueueURLPattern = "%s/%s/%s"
+var sqsServiceName = "sqs"
+var fmtCopilotResourceNamePrefix = "%s-%s-%s-"
```

Comment:
The `aws/*` pkgs are meant to be low-level wrappers around the AWS SDK and _shouldn't_ know anything about Copilot concepts.

So their main purpose is to:
1. Isolate dependencies.
2. Write simplifying methods that chain multiple API calls together.

Can we remove any error or copilot related variables from here? Instead a higher level package such as `describe` should be responsible for it.